src/test/scala/edu/ie3/mobsim/MobilitySimulatorSpec.scala

```diff
@@ -308,32 +308,109 @@ class MobilitySimulatorSpec extends UnitSpec with MobilitySimulatorTestData {
       }
     }
 
-    "get time until next event" in {
-      val mobilitySimulator = mobSim()
-
-      val getTimeUntilNextEvent =
-        PrivateMethod[Long](Symbol("getTimeUntilNextEvent"))
+    "get time until next departure" in {
+      val getTimeUntilNextDeparture =
+        PrivateMethod[Option[Long]](Symbol("getTimeUntilNextDeparture"))
 
       val ev2: ElectricVehicle =
         evChargingAtSimonaWithStation.copy(departureTime =
           givenSimulationStart.plusMinutes(45)
         )
 
       val cases = Table(
-        ("evs", "expectedNextEvent"),
-        (Seq(arrivingEv), 3600),
+        ("evs", "expectedNextDeparture"),
+        (Seq(departingEv), 3600),
```

Comment:
Maybe for consistence 3600L?

Reply:
In my understanding the L will indicates to handle the value as Long and not as Int. Since at 3600 there is no danger to have rounding problems due to an overflow of the value, this is fine but there is no harm to use L in addition. May @sebastian-peter have any preference?

Reply:
I once added the `L` everywhere, but recently I've found leaving it out more readable. It's only rarely necessary where Scala does not recognize that a Long value is needed, but turns the literal into an Int. But here it's definitely not the case, otherwise MobSim would not compile.